servers/rendering/renderer_viewport.cpp

```diff
@@ -276,6 +276,8 @@ void RendererViewport::_draw_viewport(Viewport *p_viewport) {
 		}
 	}
 
+	bool can_draw_3d = RSG::scene->is_camera(p_viewport->camera) && !p_viewport->disable_3d;
```

Comment:
```suggestion
	bool can_draw_3d = !p_viewport->disable_3d && RSG::scene->is_camera(p_viewport->camera);
```
Cheapest check first